### PR TITLE
Issue / Learn Css / Image Container

### PR DESCRIPTION
**Pull request by dncsvr**
Learn how to create style for an image within a container

## Tasks

- [x] Sample image container with fixed size
- [x] Sample image container with variable size
- [x] Documentation

### CODE REVIEW

**Review by cihandeniz — DISMISSED** [review]

File: css/image-container/image-container.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+}
+```
+
+With this way, the image fully matches the dimensions of the container, 
+however the aspect ratio of the image may be lost. We can alter the _object-fit_ 
+property to protect the aspect ratio to overcome that issue. 
+
+#### Margin/Padding Issue
+
+A problem occurs with this approach is, if we want some margin or padding for
```
Comment (alptuncs) [inline css/image-container/image-container.md]: "A problem that occurs with this approach is, ..." veya "A problem occurs with this approach, ..." daha doğru olur gibi geldi bana

File: css/image-container/image-container.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+If we have a fixed sized container we can use full width and height.
+
+```css
+.image-container-fixed img {
+    width: 100%;
+    height: 100%;
+}
+```
+
+With this way, the image fully matches the dimensions of the container,
```
Comment (alptuncs) [inline css/image-container/image-container.md]: "This way, ..." veya "With this, ..." olabilir

File: css/image-container/image-container.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+```css
+.image-container-fixed img {
+    width: 100%;
+    height: 100%;
+}
+```
+
+With this way, the image fully matches the dimensions of the container, 
+however the aspect ratio of the image may be lost. We can alter the _object-fit_ 
+property to protect the aspect ratio to overcome that issue.
```
Comment (alptuncs) [inline css/image-container/image-container.md]: "... that issue." yerine "... this issue." daha iyi olur gibi geldi

File: css/image-container/image-container.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+#### Margin/Padding Issue
+
+A problem occurs with this approach is, if we want some margin or padding for 
+the image, the image will shift inside the container. We can set padding of the 
+parent container and reduce its size accordingly but in some cases it may 
+not be the best solution.
+
+A way to solve this problem is to use ```@container``` queries. Those 
+queries can provide size information about a html element. If we get the size
```
Comment (alptuncs) [inline css/image-container/image-container.md:33]: "Those queries ..." yerine "These queries ..." daha iyi olur bence

**Review by alptuncs — APPROVED** [review]

**Pull request merged**